packages/device_info/lib/device_info.dart

```diff
@@ -116,6 +117,9 @@ class AndroidDeviceInfo {
   /// The type of build, like "user" or "eng".
   final String type;
 
+  ///false, if device is being simulated/emulated
```

Comment:
Nits:

- space after `///`
- "False" or `false` (i.e. wrap in backticks if referring to a Dart keyword).

Reply:
Style:

```dart
/// `false` if the application is running in an emulator, `true` otherwise.
```